dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/hibernate/HibernateUserStore.java

```diff
@@ -99,6 +99,11 @@ private Query getUserQuery( UserQueryParams params, boolean count )
             hql += hlp.whereAnd() + " uc.disabled = :disabled ";
         }
 
+        if ( params.isNot2FA() )
+        {
+            hql += hlp.whereAnd() + " uc.secret is null ";
+        }
+

```

Comment:
Should we also add a test for "twoFA" field here?

Reply:
Not sure what purpose that would serve?